### PR TITLE
Standalone fix

### PR DESCRIPTION
**Pull request by NoahRJohnson**
No one else should have to suffer to build pyvxl locally.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,6 +29,18 @@ if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
     include_directories(${VXL_CORE_INCLUDE_DIR})
     include_directories(${VXL_VCL_INCLUDE_DIR})
 
+    # for pyvxl
+    include_directories(${VXL_SOURCE_DIR})
+    include_directories(${VXL_SOURCE_DIR}/core)
+    include_directories(${VXL_SOURCE_DIR}/vcl)
```
Comment (decrispell) [inline CMakeLists.txt]: Are lines 34 and 35 needed?  Seems redundant with lines 29 and 30

File: README.md [hunk 2]
```diff
@@ -13,8 +13,37 @@ Python wrapper for vxl using pybind11
 pyvxl can be build standalone or as a module of an existing project.
 
 ## Standalone
+
+Whatever environment (your host or a docker container) you build pyvxl in will require these dependencies:
+- g++
+- Python 3 development libraries (e.g. `python3-devel`)
+- GeoTIFF development libraries (e.g. `libgeotiff-devel`)
+
+If you're on your host, it's recommended you do this in a virtualenv so you don't mess with your
```
Comment (decrispell) [inline README.md]: missing the end of the sentence?

Comment (NoahRJohnson) [reply README.md]: Thanks that sentence was replaced by the virtualenv section below.

File: CMakeLists.txt (hunk 1 above)
Comment (NoahRJohnson) [reply CMakeLists.txt]: You're right, they were redundant.

**Pull request merged**